Author: Unrud <unrud@outlook.com>
Description: hide the "Upload Highscores" setting, because the server is down
Last-Update: 2019-05-10
--- a/src/Settings.py
+++ b/src/Settings.py
@@ -138,7 +138,7 @@
       ConfigChoice(engine.config, "game",  "language"),
       ConfigChoice(engine.config, "game",  "leftymode", autoApply = True),
       ConfigChoice(engine.config, "game",  "tapping", autoApply = True),
-      ConfigChoice(engine.config, "game",  "uploadscores", autoApply = True),
+      #ConfigChoice(engine.config, "game",  "uploadscores", autoApply = True),  # server is down
     ]
     gameSettingsMenu = Menu.Menu(engine, gameSettings + applyItem)
 
